docs/user-guide/managing-deployments.md

```diff
@@ -85,8 +85,8 @@ NAME           CLUSTER-IP   EXTERNAL-IP   PORT(S)      AGE
 my-nginx-svc   10.0.0.208                 80/TCP       0s
 ```
 
-With the above commands, we first create resources under docs/user-guide/nginx/ and print the resources created with `-o name` output format 
-(print each resource as resource/name). Then we `grep` only the "service", and then print it with `kubectl get`. 
+With the above commands, we first create resources under docs/user-guide/nginx/ and print the resources created with `-o name` output format
+(print each resource as resource/name). Then we `grep` only the "service", and then print it with `kubectl get`.
```

Comment:
Please do not change the default markdown format such as adding an unnoticed short blank at the end of line.